main.go

```diff
@@ -206,6 +206,16 @@ func main() {
 		reason = fmt.Sprintf("Commit message `%s` at commit %s doesn't follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) style.", badCommit.message, badCommit.url)
 	}
 
+	log.Printf("Title is valid: %s", strconv.FormatBool(isTitleValid))
+	log.Printf("Length of body: %d", len(body))
+	log.Printf("Count of issues: %d", len(issueMentions))
+	log.Printf("Too may changes in PR: %s", strconv.FormatBool(hasTooManyChanges))
```

Comment:
Typo here, should be `many` if I interpreted correctly.

Reply:
Done